sharepoint/3.1/modules/ROOT/pages/sharepoint-connector-reference.adoc

```diff
@@ -1,1320 +1,1365 @@
+| Disable SSL certificate validation a| Boolean |  Disables the CN (Common Name) check so that the server won't throw an exception if the HTTPS certificate is not signed by a trusted partner.
+
+NOTE: Disabling the CN check is not recommended for production environments.  |  false |
+| Host a| String |  Hostname for the proxy server. |  |
+| Port a| Number |  Port number for the proxy server. |  |
+| Username a| String |  Username to use for authenticating against the proxy server |  |
+| Password a| String |  Password to use for authenticating against the proxy server |  |
+| NTLM Domain a| String |  Domain to use for authenticating against the proxy server|  |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+|===
+[[sharepoint-onpremise_ntlm]]
+===== NTLM
 
-* proxy
-* ntlm-proxy | Settings for the proxy server that the connector is going to send data through. | |
-| Reconnection a| Reconnection | When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. | |
+
+====== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
 |===
+| Name | Type | Description | Default Value | Required
+| Domain a| String |  Domain name to use for NTLM authorization |  | x
+| Ntlm Username a| String |  Username to log in with. Do not include the domain name. |  | x
+| Ntlm Password a| String |  Password NTLM authorization. |  | x
+| Site Url a| String |  URL for the target SharePoint instance |  | x
+| Disable SSL certificate validation a| Boolean |  Disables the CN (Common Name) check so that the server won't throw an exception if the HTTPS certificate is not signed by a trusted partner.
+
+NOTE: Disabling the CN check is not recommended for production environments.  |  false |
+| Host a| String |  Hostname for the proxy server. |  |
+| Port a| Number |  Port number for the proxy server. |  |
+| Username a| String |  Username to use for authenticating against the proxy server |  |
+| Password  a| String |  Password to use for authenticating against the proxy server |  |
+| NTLM Domain a| String |  Domain to use for authenticating against the proxy server|  |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+|===
+[[sharepoint-onpremise_security-token]]
+===== Security Token
 
-[[sharepoint-onpremise_kerberos]]
-=== Kerberos
 
-==== Parameters
+Deprecated the SecurityTokenConnectionStrategy connection provider because the BasicConnectionProvider already obtains the security token
```

Comment:
```suggestion
The Security Token connection strategy connection provider is deprecated because the Basic Connection provider already obtains the security token.
```